src/dashboard/src/settings/common.py

```diff
@@ -340,3 +360,15 @@ def get_env_variable(var_name):
 TEXTAREA_WITH_HELP_ATTRS = {'rows': '4', 'class': 'span11 has_contextual_help'}
 INPUT_ATTRS = {'class': 'span11'}
 INPUT_WITH_HELP_ATTRS = {'class': 'span11 has_contextual_help'}
+
+SHIBBOLETH_REMOTE_USER_HEADER = 'HTTP_EPPN'
+SHIBBOLETH_ATTRIBUTE_MAP = {
+    # Automatic user fields
+    'HTTP_GIVENNAME': (False, 'first_name'),
+    'HTTP_SN': (False, 'last_name'),
+    'HTTP_MAIL': (False, 'email'),
+    # Entitlement field (which we handle manually)
+    'HTTP_ENTITLEMENT': (True, 'entitlement'),
+}
+# If the user has this entitlement, they will be a superuser/admin
```

Comment:
We don't need to worry about whether the user has the normal entitlement (i.e. `preservation-user`) - nginx will not let them in at all without this entitlement. We just look at the admin flag to determine whether to make them a superuser.